src/components/SiteNav/styles.css

```diff
@@ -4,18 +4,68 @@
 	font-weight: bold;
 }
 
-#site-nav ul {
+#site-nav a { color: white; }
+
+#logo-rounded {
+	border-radius: var(--size-2);
+}
+
+#nav-menubar {
 	gap: var(--size-2);
 	align-items: center;
 	padding: var(--size-3);
 }
 
-#site-nav a { color: white; }
+#nav-menubar .mobile-only {
+	display: none;
+}
 
-#logo-rounded {
-	border-radius: var(--size-2);
+@media (max-width: 500px) {
+	#nav-menubar {
+		justify-content: space-between;
+	}
+
+	#nav-menubar .mobile-only {
+		display: block;
+	}
+
+	#nav-menubar .desktop-only {
+		display: none;
+	}
+}
+
+#nav-menu-button {
+	display: none;
+	background-color: transparent;
+}
+
+#nav-menu-button .hidden {
+	display: none;
 }
 
 @media (max-width: 500px) {
-	#site-nav ul { flex-direction: column; }
+	#nav-menu-button {
+		display: block;
+	}
+}
+
+#mobile-menu:popover-open {
```

Comment:
Please move over these declarations before the media queries.

As CSS rules are executed in sequence, it is a good practice to put all media queries at the bottom as they will override the previous rules and not get overridden by then.